src/components/profile-sidebar/profile-sidebar.module.css

```diff
@@ -0,0 +1,8 @@
+.vl {
```

Comment:
Avoid naming anything with single or double letters or any name that is vague, what's vl ??